examples/04_fuzzy_joining.py

```diff
@@ -397,9 +397,9 @@
 
 # We will test four possible values of match_score:
 params = {
-    "joiner-1__match_score": [0.2, 0.9],
-    "joiner-2__match_score": [0.2, 0.9],
-    "joiner-3__match_score": [0.2, 0.9],
+    "joiner-1__match_score": [0.1, 0.9],
+    "joiner-2__match_score": [0.1, 0.9],
```

Comment:
I can't comment on it but you need to update the description regarding the best `match_score` parameter here and in other places of the example.
"The grid searching gave us the best value of 0.5 for the parameter"